storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/delete/PreparedDeleteByQuery.java

```diff
@@ -131,6 +108,35 @@ public Completable asRxCompletable() {
         return RxJavaUtils.createCompletable(storIOSQLite, this);
     }
 
+    @NonNull
+    @Override
+    protected Interceptor getRealInterceptor() {
+        return new RealInterceptor();
```

Comment:
A agree that variant with standalone class will be more readable. 
Also if we will decided some days to make it public, there will be less things to change=)